### PR TITLE
Mailer queue implemented

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/5269

*Description of changes:*

- Added mailer queue.No actual logic. Only the setup is inlcuded in this PR

**Let me know if the name mailer queue makes sense.Not 100% satisfied with it.So happy to receive suggestions and change it.**
*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/account-mailer/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+
+# Application Name
+
+BC Registries Names Examination, research and approval system API
```
Comment (sumesh-aot) [inline queue_services/account-mailer/README.md:7]: Needs to be changed. Looks like pay also have this copy error. :)

File: queue_services/account-mailer/q_cli.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        await sc.close()
+        await nc.close()
+
+    # Connection and Queue configuration.
+    def nats_connection_options():
+        return {
+            'servers': os.getenv('NATS_SERVERS', 'nats://127.0.0.1:4222').split(','),
+            'io_loop': loop,
+            'error_cb': error_cb,
+            'name': os.getenv('NATS_CLIENT_NAME', 'entity.filing.tester')
```
Comment (sumesh-aot) [inline queue_services/account-mailer/q_cli.py:56]: Can we put the defaults as we discussed, starting with account.*. This is a TODO for pay q listener as well.

File: queue_services/account-mailer/q_cli.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        await nc.connect(**nats_connection_options())
+        await sc.connect(**stan_connection_options())
+
+        # register the signal handler
+        for sig in ('SIGINT', 'SIGTERM'):
+            loop.add_signal_handler(getattr(signal, sig),
+                                    functools.partial(signal_handler, sig_loop=loop, sig_nc=nc, task=close)
+                                    )
+
+        payload = {
```
Comment (sumesh-aot) [inline queue_services/account-mailer/q_cli.py:84]: Also the payload need change. But can be done later once it's finalized

**Review by sumesh-aot — APPROVED** [review]
Added few minor comments

**Review by nitheesh-aot — APPROVED** [review]

**Review by mengdong19 — APPROVED** [review]

File: queue_services/account-mailer/q_cli.py (hunk 3 above)
Comment (saravanpa-aot) [reply queue_services/account-mailer/q_cli.py:84]: i will do this in the next PR.

File: queue_services/account-mailer/README.md (hunk 1 above)
Comment (saravanpa-aot) [reply queue_services/account-mailer/README.md:7]: i will do it in next PR

File: queue_services/account-mailer/q_cli.py (hunk 2 above)
Comment (saravanpa-aot) [reply queue_services/account-mailer/q_cli.py:56]: i will do it in next PR

**Pull request merged**